### PR TITLE
Firmware flasher: classic build toggle - moved to build configuration header

### PR DESCRIPTION
**Pull request by limonspb**
Firmware flasher:
- Classic build toggle - moved to build configuration header
- Hide build configuration fields instead of disabling them because they are not used in classic mode.
- Addressed this comment: https://github.com/betaflight/betaflight-configurator/pull/3089#issuecomment-1321098880

image

### CODE REVIEW

**Review by KarateBrot — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
image

While your at it, please also give the help icon a margin-top of 10px.

**Comment by limonspb** [comment]
> image
> 
> While your at it, please also give the help icon a margin-top of 10px.

I'll do something with them in the next PR. They also behave weirdly when you make the window small.

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
